html-forms/<!DOCTYPE html>.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML-Forms</title>
+  </head>
+  <body>
+    <h1><strong> Coding Form! </strong></h1>
+    <label for="Email"> Email:</label>
+    <div>
+      <input type="text" id="Email1" name="Email" required />
+    </div>
```

Comment:
This won't link your `label` to your `input` for email. What does the `for` attribute connect to on your `input`?